.vscode/launch.json

```diff
@@ -1,9 +1,60 @@
 {
-  // Use IntelliSense to learn about possible attributes.
-  // Hover to view descriptions of existing attributes.
-  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
+    {
+      "name": "Python: test metadata loader",
+      "type": "python",
+      "request": "launch",
+      "module": "pytest",
+      "cwd": "${workspaceFolder}/tools/metadata_loader"
+    },
+    {
+      "name": "datasource list",
```

Comment:
Since we have just one launch.json would be good to identify these actions as being form the metadata loader something liek adding [LOADER] to the name.

Reply:
ok